blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -83,20 +129,89 @@ const MediumPromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <MediumPromoPresentation content={content} {...customFields} />;
-};
-
-const MediumPromo = ({ customFields }) => {
-	const { isAdmin, arcSite } = useFusionContext();
-	const shouldLazyLoad = customFields?.lazyLoad && !isAdmin;
 	if (shouldLazyLoad && isServerSide()) {
 		return null;
 	}
-	return (
+
+	const hasAuthors = showByline ? content?.credits?.by : null;
+	const contentDescription = showDescription ? content?.description?.basic : null;
+	const contentHeading = showHeadline ? content?.headlines?.basic : null;
+	const contentUrl = content?.websites?.[arcSite]?.website_url;
+	const imageSearchField = imageOverrideURL ? "imageOverrideURL" : "imageURL";
+	const promoImageURL = imageOverrideURL || getImageFromANS(content);
+	const contentDate = content?.display_date;
+	const formattedDate = Date.parse(contentDate)
+		? localizeDateTime(new Date(contentDate), dateTimeFormat, language, timeZone)
+		: "";
+
+	const MediaImage = () =>
+		showImage ? (
+			<Conditional component={Link} condition={contentUrl} href={contentUrl}>
+				<Image
```

Comment:
We should be using resizer v2 now for rendering images

Reply:
@matthewroach : I agree but think we should scope this in another ticket. We need to do this for all eight promo blocks and need a consistent way of getting image `auth` values from ANS story data (perhaps similiar to how we get image URLs with `getImageFromANS`). I'd be happy to create ticket(s) where the AC is to:

1. Create a utility to find `auth` tokens for story images.
2. Implement resize logic for all promo blocks, using the utility to get the `auth` token.

I believe this will help us to stay Agile as resizing isn't specifically cited in [this ticket's AC](https://arcpublishing.atlassian.net/browse/TMEDIA-742).

Please let me know what you think. Thanks!